### DISCUSSION TITLE
you want a pull request, right?🥳

### DISCUSSION THREAD

**Discussion by RyoheiKokura** [NAIST-SE/AttractingContributors#169] [Show and tell]
hi dear all.
my hobby is send to pull request :^)

Please write your project URL. I'm okay without comment:^)
As soon as I finish preparing, I will send you pull request!

No worries, you don't need to send me pull request! 
It's OK if you do it or not.
because my hobby is send to pull request :^)

by https://github.com/RyoheiKokura/AttractingContributors_PullRequest

**Comment by ElefunAi** [comment]
That's very helpful for me! Thank you.
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Reply by RyoheiKokura** [reply]
I did it!
Sorry for the late reply!

**Comment by YPanY** [comment]
Please help me by adding your favorite programming language or framework here👇 thank you RyoheiKokura!
https://github.com/Pan-NLP-NAIST/AttractingContributors-programming_language/blob/main/README.md

**Reply by RyoheiKokura** [reply]
I did it!
Sorry for my late reply!

**Comment by Gavin-23** [comment]
Added!
Please help me too~~
Thanks 😊
https://github.com/Gavin-23/AttractingContributors-Music

**Reply by RyoheiKokura** [reply]
I did it!!!!
Sorry for my late reply!!!!

**Comment by xincanfeng** [comment]
added!
please tell me your favorite color!
https://github.com/FengXincan/AttractingContributors-color

**Reply by RyoheiKokura** [reply]
I did it!

**Comment by veekay7** [comment]
Just added a memorable quote to ur repo.
Please tell me the country/countries you love to visit in the future.
Cheers.

https://github.com/VantelinLuca/AttractingContributors-WishCountry

**Reply by RyoheiKokura** [reply]
I'm sorry for getting back to you so late!!!!!!!
